linode/helper/framework_conversion.go

```diff
@@ -114,6 +114,19 @@ func GenericSliceToList[T any, U attr.Value](
 	return listResult
 }
 
+func GetListOfSingleObjectValueFromMap(
```

Comment:
nitpick: It might make sense to rename this helper to something like `MapToSingleObjList`. A documentation comment might also be handy here :slightly_smiling_face:

Reply:
Fixed. Thanks!